examples/simple_parse.c

```diff
@@ -103,7 +107,11 @@ void print_json_array(json_t *element, int indent) {
     size_t size = json_array_size(element);
     print_json_indent(indent);
 
-    printf("JSON Array of %ld element%s:\n", size, json_plural(size));
+#if defined(__STDC_VERSION__) && __STDC_VERSION__ >= 201112L
+    printf("JSON Array of %zd element%s:\n", size, json_plural(size));
+#else
+    printf("JSON Array of %lld element%s:\n", (long long)size, json_plural(size));
```

Comment:
If `%lld` and the `(long long)` cast works for all compilers, then I think it would be better to just do that, and omit the other 4 lines of code.

Reply:
Fine with that. Let me fix it.

Reply:
Thanks. The rest looks good to me, but I have no authority here :)